livre/core/render/FrameInfo.cpp

```diff
@@ -23,9 +23,11 @@ namespace livre
 {
 
 FrameInfo::FrameInfo( const Frustum& frustum_,
-                      const uint32_t frameId_ )
+                      const uint32_t frameId_,
+                      const uint32_t frameCounter_ )
```

Comment:
Frame ID and frame counter? This looks fishy
Can you explain why you need this?